blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpRollupHandlerWithESIntegrationTest.java

```diff
@@ -135,7 +135,7 @@ public void testUnknownUnit() throws Exception {
                 baseMillis + 86400000,
                 1600);
         Assert.assertEquals(data.getData().getPoints().size(), 0);
-        Assert.assertEquals(data.getUnit(), "unknown");
+        Assert.assertNull(data.getUnit());
```

Comment:
@tilogaat We do stamp units as `UNKNOWN` when we serialize the metrics before sending back the response. I think that should be fine, unless you have really strong opinions about it. https://github.com/rackerlabs/blueflood/pull/464/files#diff-386fab66e274547563dbf35dafbd3099R72